test/cli/stat.t/run.t

```diff
@@ -0,0 +1,37 @@
+No files are shown when running `stat` in a non-existent directory:
+
+  $ ../index_fsck.exe stat ../data/non-existent-store
+  >> Getting statistics for store: `../data/non-existent-store'
+  
+  {
+    "entry_size": "40.0 B",
+    "files": {}
+  }
+
+Running `stat` on an index after 10 merges:
+
+  $ ../index_fsck.exe stat ../data/random > log.txt 2>&1
+  $ sed -re 's/"lock": "[0-9]+"/"lock": "<PID>"/g' log.txt
```

Comment:
`sed --r` does not work on macOS (`sed: illegal option -- r`), but `sed -E ..` does.